server.go

```diff
@@ -40,19 +40,14 @@ func StartServer(user, pass, jiraHost string, fc FieldConfig) error {
 	r.GET("/api/milestones/:key", gc.getMilestoneGraph)
 
 	// this roundabout way of implementing FileFromFS is needed because of the index.html special case: https://github.com/gin-gonic/gin/issues/2654
+	if templates, err := template.ParseFS(distFS, "dist/*.html"); err != nil {
+		log.Panic(err)
+	} else {
+		r.SetHTMLTemplate(templates)
+	}
```

Comment:
Since the `true` case panics, I suggest out-denting the 'else' block and eliminating the `else`.

Reply:
Sure 👍 this was a little of the style from below with `fs.Sub`, and a bit more of an overzealous tendency to minimize the scope of variables 😅 https://github.com/andrei-m/jira-graph/pull/28/commits/dcdab61b1dbbd1c096527f728a2e1a29072cbc17